src/SDKS/erc20/index.ts

```diff
@@ -125,7 +129,7 @@ export namespace CryptoWallet.SDKS.ERC20 {
      * @param rawTx
      * @param network
      */
-    broadcastTx(
+    public broadcastTx(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after public instance fields.](https://app.codacy.com/app/atlascity/cryptowallet-js/pullRequest?prid=3649440)